package.json

```diff
@@ -132,7 +144,12 @@
     "exclude": [
       "src/database/*",
       "src/config/*",
-      "src/utils/logger.ts"
+      "src/utils/logger.ts",
+      "src/utils/cloud.ts",
+      "src/middleware/fileUpload.ts",
+      "src/helper/excOperation.ts",
+      "src/helper/productQueries.ts",
+      "src/utils/sendEmail.ts"
     ]
```

Comment:
why this file removed

Reply:
i removed this because it was bringing my test down and it requires internet to be tetsted